packages/dds/tree/prettier.config.cjs

```diff
@@ -0,0 +1,40 @@
+// TODO: Use shared config in build-common once version with config has been published.
```

Comment:
Is this copied from one in an unreleased version of build-common? Would be good to explicitly state that if it is.